common/service/dynamicconfig/config.go

```diff
@@ -75,9 +75,10 @@ func (c *Collection) logValue(
 	cmpValueEquals func(interface{}, interface{}) bool,
 ) {
 	loadedValue, loaded := c.keys.LoadOrStore(key, value)
-	if !loaded || !cmpValueEquals(loadedValue, value) {
-		c.logger.Debug("Get dynamic config",
-			tag.Name(key.String()), tag.Value(value), tag.DefaultValue(defaultValue))
+	// If not loaded before, then print the log for the first loaded 
+	if !loaded {
```

Comment:
I am a little bit confused by the logic here. It seems like we want to emit a log when the value is first loaded and when the value changes. But the values in `keys` was never updated after the first load, so we will keep printing logs if the values changes. If we fix that, we should be able to emit log in both cases without generating too many logs?

Reply:
I misunderstood the code here because of the names. Just updated the PR, does it make more sense to you now?

Reply:
👍  Yup, looks good to me! 

nit: Log messages should begin with capital letter?

Reply:
yeah for sure. Thank you so much for reviewing it.